src/components/ControlPanel.jsx

```diff
@@ -1,33 +1,85 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchGuide } from '../ducks/field-guide';
+import { toggleDialog } from '../ducks/dialog';
+import FieldGuide from '../components/FieldGuide';
 
-export default function ControlPanel() {
-  return (
-    <section className="control-panel">
-      <div className="control-panel__header">
-        <h4 className="primary-label">Subject info</h4>
-        <span>&#x025B8;</span>
-      </div>
-      <hr />
-      <div className="control-panel__buttons">
-        <div>
-          <span className="primary-label">Name</span>
-          <span className="body-font">ENA NS 78 0117</span>
-        </div>
-        <div>
-          <span className="primary-label">Attribution</span>
-          <span className="body-font">Library of the Jewish Theological Seminary</span>
+class ControlPanel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.toggleFieldGuide = this.toggleFieldGuide.bind(this);
+  }
+
+  componentWillMount() {
+    this.props.dispatch(fetchGuide());
```

Comment:
Nyuuurrr... after our experience with ASM, I'm starting to favour making all `fetch()` calls either
1. centralised in a main component (or root component), or better yet
2. centralised in a main lifecycle call (e.g. chain it in `fetchProject()` )

On the other hand, I can see the argument for keeping this as is: _"The Field Guide is only ever required when the Classifier Page is accessed, so there's no reason to load the FG when a user is simply looking at the site for its About page, dude."_

On the third flailing appendage, I can see a counter-argument to that argument, in which I say, _"Yeah but 1. this should be in the Classifier Page container not the Control Panel compo, 2. now you're making a call for the Field Guide EVERY time the Classifier Page loads which is a waste of resources, and 3. don't call me dude, man."_

Reply:
No, I definitely want to have a sort of wrapper or root component that will collect all the resources the app needs. Should we have a sort of loader as in ASM or just return null until some sort of initialized prop is set? There are many ways to do this.